ui/raidboss/data/07-dt/dungeon/yuweyawata.ts

```diff
@@ -31,6 +31,7 @@ const triggerSet: TriggerSet<Data> = {
         text: {
           en: 'Away from falling pillar',
           de: 'Weg von der fallenden Säule',
+          fr: 'Éloignez-vous du pilier',
```

Comment:
https://youtu.be/D0mGNw_Ps6Q?si=I61co2e8PerbEp9M&t=14
![image](https://github.com/user-attachments/assets/d052aa1b-e52b-49ad-a993-537f5ded34d5)

It's better if it means where the pillar falls.

Reply:
You're right.